.github/workflows/sanitized_ci.yml

```diff
@@ -157,7 +157,7 @@ jobs:
       - name: Self tests (V compiled with -fsanitize=address)
         run:
           ./v -cflags -fsanitize=address -o v cmd/v &&
-          ASAN_OPTIONS=detect_leaks=0 ./v -cc tcc test-self -asan-compiler
+          ASAN_OPTIONS=detect_leaks=0 ./v -cc tcc -cflags -fsanitize=address test-self -asan-compiler
```

Comment:
`ASAN_OPTIONS=detect_leaks=0 ./v -cc tcc test-self -asan-compiler`
Here, we are testing not that the tests can pass with these cflags, but that `v` itself, compiled with `-cflags -fsanitize=address` can compile the tests cleanly.

Note, that the compiler is `-cc tcc`, which does not support `-fsanitize=address` (it just ignores that flag).